pywbem/cim_operations.py

```diff
@@ -6399,12 +6445,23 @@ def ModifyInstance(self, ModifiedInstance, IncludeQualifiers=None,
             cannot rely on it being implemented by WBEM servers.
 
           PropertyList (:term:`string` or :term:`py:iterable` of :term:`string`):
-            An iterable specifying the names of the properties (or a string
-            that defines a single property) to be modified (case independent).
+            This parameter defines which properties are designated to be
+            modified.
 
-            An empty iterable indicates to modify no properties.
+            It is an iterable specifying the names of the properties, or a
```

Comment:
Suggest that 'It is an iterable..'. be changed to The PropertyList is ...

Reply:
I don't like "The {parameter-name} is ...". I have changed it to "This parameter is ...", consistent with the sentence before the one you commented on.